Src/FluentAssertions/Types/TypeSelector.cs

```diff
@@ -183,6 +183,26 @@ public TypeSelector ThatAreNotClasses()
         return this;
     }
 
+    /// <summary>
+    /// Determines wheter the type is an interface; that is, not a class or a value type
```

Comment:
🔧 It doesn't determine whether a type is an interface. It filters them. Same with the other methods.